chains/meter.json

```diff
@@ -0,0 +1,21 @@
+{
+  "alias": "meter",
+  "decimals": 18,
+  "explorer": {
+    "browserUrl": "https://scan.meter.io/"
+  },
+  "id": "82",
+  "name": "Meter",
```

Comment:
To note: it has fixed gas price 100 gwei. See https://docs.meter.io/wallet-setup/specifying-gas-gwei-for-transactions. 

I checked that `eth_gasPrice` returns 100 gwei always. So it shouldn't be problem for our applications.

Reply:
Also better to check important things to pay attention: https://docs.meter.io/developer-documentation/introduction#id-8.-important-things-to-pay-attention-to